libse/RegexUtils.cs

```diff
@@ -176,8 +176,8 @@ public static bool IsValidRegex(string testPattern)
             }
             try
             {
-                Regex.Match(string.Empty, testPattern);
-                return true;
+                return Regex.IsMatch(string.Empty, testPattern);
+                //return true;
```

Comment:
The old code is correct - it does not need to be a match but only not throw an `ArgumentException`.
Luckily a unit test caught this: https://ci.appveyor.com/project/SubtitleEdit/subtitleedit/build/1.0.7288